lib/please_cli/please_cli/update_dependencies.py

```diff
@@ -43,8 +81,70 @@
         please_cli.config.NIX_BIN_DIR + 'nix-shell',
         ),
     )
+@click.option(
+    '--git',
+    required=True,
+    default='git',
+    help='Path to git command (default: git).',
+    )
+@cli_common.cli.taskcluster_options
 @click.pass_context
-def cmd(ctx, project, nix_shell):
+def cmd(ctx,
+        project,
+        branch_to_push,
+        git_url,
+        git_user_email,
+        git_user_name,
+        nix_shell,
+        git,
+        taskcluster_secret,
+        taskcluster_client_id,
+        taskcluster_access_token,
+        ):
+
+    # if no branch is provided we assume we don't want to push changed anywhere
+    # we just run the update
+    if branch_to_push is None:
+        return run_update(project, nix_shell, os.getcwd())
+
+    # a directory where we will cloned release-services
+    root_dir = tempfile.mktemp(prefix='release-services-')
```

Comment:
`mktemp` is deprecated, can you use `mkdtemp` instead. It will create the directory, but you still will be able to clone to it, because it's empty.
Also, do we care about cleaning up after we are done? In CI it doesn't really matter, but could become a bit annoying for local testing. We can live with this for now in either case.